service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -276,4 +286,67 @@ private void buildCoreItem(String instanceId, String barcode, NodeList nodes, Ma
         catalogItems.put(barcode, itemData);
     }
 
+    @Override
+    public FeesFines getFeesFines(String uin) {
+        String path = "patron/account/";
+        String additional = "&includeLoans=false&includeCharges=true&includeHolds=false";
+        String url = String.format("%s%s%s?apikey={apikey}%s", getAPIBase(), path, uin, additional);
```

Comment:
I think this approach is cleaner.

```
String urlTemplate = "{baseUrl}/patron/account/{uin}?apiKey={apiKey}?includeCharges=true";
Map<String, String> uriVariables = new HashMap<>();
uriVariables.put("baseUrl", getAPIBase());
uriVariables.put("uin", uin);
uriVariables.put("apiKey", apiKey);
JsonNode node = restTemplate.getForObject(urlTemplate, JsonNode.class, urlVariables);
```

With Java 9+
```
Map<String, String> uriVariables = Map.of(
  "baseUrl", getAPIBase(),
  "uin", uin,
  "apiKey", apiKey
);
```

Clearly subjective and just want to propose.

Reply:
I was considering using only the substitution, but my focus is on also using the current log.
I could, however, create a separate string for printing the log.

As far Java 9+, are we moving this onto JDK 11.
I thought this project is still (currently) stuck with JDK 8 at the moment.

Reply:
Yeah, we are still on Java 8. Just showing it would be nice to utilize Java 9+. For logging should we try to use logging RestTemplate affords instead of our own logging statement?

```
logging.level.org.springframework.web.client.RestTemplate=DEBUG
```

If this prints a reasonable log without the apiKey, I vote to avoid our own logging.